internal/operands/metrics/resources.go

```diff
@@ -164,6 +164,22 @@ func getAlertRules() ([]promv1.Rule, error) {
 				componentAlertLabelKey:    componentAlertLabelValue,
 			},
 		},
+		{
+			Alert: "VirtualMachineCRCErrors",
+			Expr:  intstr.FromString("kubevirt_ssp_vm_rbd_volume{volume_mode=\"Block\", rxbounce_enabled=\"false\"} > 0"),
```

Comment:
Is the issue specific to `Block` PVCs?

Reply:
yes

Reply:
adding source here for future ref https://github.com/kubevirt/kubevirt/pull/9741#issuecomment-1612535963